package.json

```diff
@@ -24,8 +24,8 @@
     "build": "yarn install-if-deps-outdated && yarn clean-client && yarn build-client && yarn build-extensions",
     "build-client": "gulp",
     "build-extensions": "webpack --config ./webpack.config.extensions.js",
-    "build-production-client": "NODE_ENV=production BABEL_ENV=production yarn build-client",
-    "build-production": "yarn distclean && yarn && gulp languages:extract && yarn build-production-client",
```

Comment:
Seems like we could hold off from these changes in this PR and just sort out the bash scripts out first.

Reply:
The second line (gulp:language) needs to change since it expects to have the POT file from a built version in my testing.